src/host/hooks/network/useGetMe.ts

```diff
@@ -30,6 +30,7 @@ export const useGetMe = () => {
         }
 
         const { data }: UserResponse = await response.json();
+        console.info('[useGetMe]', data);
```

Comment:
Will move this one into console wrapper, also, for sports connect, we should add https://babeljs.io/docs/babel-plugin-transform-remove-console/